src/app/accounts/sso.component.ts

```diff
@@ -31,4 +32,23 @@ export class SsoComponent extends BaseSsoComponent {
         this.redirectUri = window.location.origin + '/sso-connector.html';
         this.clientId = 'web';
     }
+
+    async ngOnInit() {
+        const queryParamsSub = this.route.queryParams.subscribe(async (qParams) => {
+            if (qParams.code != null && qParams.state != null) {
+                if (qParams.state.endsWith(':clientId=browser')) {
+                    window.postMessage({ type: "AUTH_RESULT", code: qParams.code, state: qParams.state }, "*");
+                }
+            }
```

Comment:
just had another thought. Could we just do this in the sso-connector.html file instead of needing to redirect back into this sso component?

Reply:
Are you thinking in a script block? @kspearrin

Reply:
Yes, there is JS there already that is parsing the query params

Reply:
Like that approach. Tested and submitted PR.